File: CoffeeBack/src/main/java/br/ufsm/csi/coffee/model/Comanda.java
package br.ufsm.csi.coffee.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Comanda {
    private int codigo;
    private String data_criada;
    private String cliente;
    private int mesa;
    private boolean ativo;
}

File: CoffeeFront/src/app/principal/comanda/comanda.ts
import { Pedido } from "../pedido";

export class Comanda{
    codigo?: number;
    data_criada?: string;
    cliente?: string;
    mesa?: number;
    ativo?: boolean;
    pedidos?: Pedido[];

    constructor(codigo?: number, data_criada?: string, cliente?: string, ativo?: boolean, mesa?: number, pedidos?: Pedido[]){
        this.codigo = codigo;
        this.data_criada = data_criada;
        this.cliente = cliente;
        this.mesa = mesa;
        this.ativo = ativo;
        this.pedidos = [];
    }
}